website/src/app/blog/sans-io/readme.mdx

```diff
@@ -2,13 +2,13 @@ import Image from "next/image";
 import Alert from "@/components/DocsAlert";
 
 At Firezone, we build secure remote access that scales, be it from your Android
-phone, MacOS computer or Linux server. At the core of these apps sits a
-connectivity library - aptly named
-[`connlib`](https://www.github.com/firezone/firezone/tree/main/rust/connlib) -
-that manages network connections and WireGuard tunnels to secure your traffic.
-After several iterations, we’ve landed on a design that we are extremely happy
-with. It gives us blazingly fast and exhaustive tests, deep customisation and
-overall high assurance that it does what we want it to do.
+phone, MacOS computer or Linux server. At the core of each app sits a
+connectivity library &mdash; aptly named
+[`connlib`](https://www.github.com/firezone/firezone/tree/main/rust/connlib)
+&mdash; that manages network connections and WireGuard tunnels to secure your
+traffic. After several iterations, we’ve landed on a design that we are
+extremely happy with. It gives us blazingly fast and exhaustive tests, deep
+customisation and overall high assurance that it does what we want it to do.
```

Comment:
This is just applying the mdash (I hate prettier formatting of MD too, but it's very helpful for the JSX components that are embedded in our MDX!)